src/pages/ConditionsList/index.tsx

```diff
@@ -226,34 +223,20 @@ export const ConditionsList: React.FC = () => {
           onClick: undefined,
         },
         {
-          href: `/split/`,
+          href: `/split/${id}`,
           text: 'Split Position',
-          onClick: () => {
-            setValue(id)
-          },
           disabled: !isConnected,
         },
         {
-          href: `/merge/`,
-          text: 'Merge Positions',
-          onClick: () => {
-            setValue(id)
-          },
-          disabled: !isConnected,
```

Comment:
Why this action was removed ?

Reply:
You can't pre-select a condition with the new UI.